src/components/profile/profile.stories.js

```diff
@@ -21,13 +21,15 @@ storiesOf('Profile', module)
     const initials = text('initials', 'JS');
     const size = select('size', OptionsHelper.sizesPortrait, OptionsHelper.sizesPortrait[0]);
     const name = text('name', 'John Smith');
+    const src = text('src', '');
 
     return (
       <ProfileWrapper
         email={ email }
         initials={ initials }
         size={ size }
         name={ name }
+        src={ src }
```

Comment:
You need to regenerate docGenInfo.json and add it to this PR so the props table picks up the `src` prop.

Reply:
Cheers for making me aware this needed doing. Understand why it needs doing and now done.